dags/analyzer_helper/discord/extract_raw_member_base.py

```diff
@@ -1,5 +1,4 @@
 class ExtractRawMembersBase:
-
     def __init__(self, guild_id: str):
```

Comment:
For the `__init__` You could either follow the instruction we gave and use `platform_id` for the input of it (as we're not discord centric and the future data sources won't have guild_id) [1], or just use no inputs and in other inherited classes you could adjust the input.

The `platform_id` is a good solution as all our data sources should have a `platform_id`, and regarding the discord you could always fetch the guild id via the `Core.platforms` collection.

[1] reference: https://rndadocs.notion.site/Analyzer-Discord-Pipeline-5009512d82a243deb35b3388e17b29c5#:~:text=The%20Base%20classes%20are%20defined%20below

Reply:
Also on method descriptions update the description to a more generalized description such as "extract platform_id related data"